README.md

```diff
@@ -1,7 +1,8 @@
-# Simple Baselines for Human Pose Estimation and Tracking (sample)
-
+# Demo of Simple Baselines for Human Pose Estimation and Tracking
 This repository contains testing code for the paper https://arxiv.org/abs/1804.06208 .  <br>
-Original repository (https://github.com/Microsoft/human-pose-estimation.pytorch)
+Original repository (https://github.com/Microsoft/human-pose-estimation.pytorch) <br>
+This repository created at Intel R&D lab 
+ <br> ![result.jpg](http://immage.biz/images/2019/04/26/SQ19.jpg) <br>
 
 # Introduction
 This demo created for quick-testing original models for mpii dataset (other datasets and models not tested) by your own images. My code draws joints, which founded by model, on your image and save it as another image. This code doesn't use any detection models, therefore searches joints by a center person of your image
```

Comment:
This repo does not have the testing code at all. It is enough to say here that it estimates only keypoints of one person, which has to be centered inside the blue rectangle.